projects/storefrontapp-e2e-cypress/cypress/integration/accessibility/tabbing-order.e2e-spec.ts

```diff
@@ -36,12 +36,19 @@ import {
   checkoutBillingAddressTabbingOrder,
 } from '../../helpers/accessibility/tabbing-order/checkout/payment-details';
 import { reviewOrderTabbingOrder } from '../../helpers/accessibility/tabbing-order/review-order';
+import { headerDesktopTabbingOrder } from '../../helpers/accessibility/tabbing-order/header';
 
 context("Tabbing order - tests don't require user to be logged in", () => {
   before(() => {
     cy.window().then(win => win.sessionStorage.clear());
   });
 
+  describe('Header - Desktop (not logged in)', () => {
+    it('should allow to navigate with tab key', () => {
+      headerDesktopTabbingOrder(config.headerDesktop);
```

Comment:
if we use `headerDesktopLoggedIn` as a name for config, please name this config here accordingly, so we will now in the future what is what. `headerDesktopAnonymousUser` or `..NotLoggedIn`